src/main/resources/META-INF/jqassistant-rules/spring-webservice.xml

```diff
@@ -0,0 +1,23 @@
+<jqassistant-rules xmlns="http://schema.jqassistant.org/rule/v1.10"
+                   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                   xsi:schemaLocation="http://schema.jqassistant.org/rule/v1.10 https://schema.jqassistant.org/plugin/jqassistant-rule-v1.10.xsd">
+
+    <concept id="spring-webservice:Endpoint">
```

Comment:
Please rename rule prefix to "spring-ws:"

Reply:
Done.